packages/htmlbars-compiler/lib/hydration-opcode-compiler.js

```diff
@@ -233,8 +222,13 @@ HydrationOpcodeCompiler.prototype.pushMorphPlaceholderNode = function() {
   this.comment();
 };
 
+HydrationOpcodeCompiler.prototype.MustacheStatement = function(mustache) {
```

Comment:
This method runs in the context of `ConcatStatement` like `<div foo="something {{bar}}">`. Due to AST update, now it is generated `concat(text("something"), mustache("bar"))` instead of  `concat(text("something"), sexp("bar"))`, so `MustacheStatement` is visited. 
I don't have a deep understanding of what is happening, but I believe that mustaches in attributes behaves different from these outside (Morphs vs AttrMorphs), so using the mustache method in that case could generate a different behavior.